test/distributed/test_c10d_functional_native.py

```diff
@@ -249,6 +249,33 @@ def func(args: List[torch.Tensor]) -> torch.Tensor:
             .run(code)
         )
 
+    @skip_if_lt_x_gpu(2)
+    def test_all_to_all_single(self) -> None:
+        self._init_process_group()
+        torch.cuda.set_device(self.device)
+
```

Comment:
curious if functional collective is supported as well on CPU backend `gloo`.

Reply:
Yes it's backend-agnostic. But yeah we should add some gloo tests.